routing/index_router.cpp

```diff
@@ -109,6 +110,28 @@ bool AllMwmsHaveRoutingIndex(Index & index, Route & route)
 
   return result;
 }
+
+pair<float, float> CalcProgressRange(Checkpoints const & checkpoints, size_t subrouteIdx)
```

Comment:
How about to create a struct with two named fields, instead of these first and second in pair?